website/snippets/_cloud-environments-info.md

```diff
@@ -42,3 +42,39 @@ By default, all environments will use the default branch in your repository (usu
 - **Deployment:** determines the branch is cloned during job executions for each environment.
 
 For more info, check out this [FAQ page on this topic](/faqs/Environments/custom-branch-settings)!
+
+
+### Extended attributes (Beta)
+
+:::important This feature is currently in beta
+
+Extended Attributes is currently in [beta](/docs/dbt-versions/product-lifecycles?) for select users and is subject to change.
+:::
+
+Extended Attributes is a feature that allows users to set a flexible [profiles.yml](/docs/core/connect-data-platform/profiles.yml) snippet in their dbt Cloud Environment settings. It provides users with more control over environments (both deployment and development) and extends how dbt Cloud connects to the data platform within a given environment.
+
+Extended Attributes is a text box extension at the environment level that overrides connection or environment credentials, including any custom environment variables. You can set any YAML attributes that a dbt adapter accepts in its `profiles.yml`.
+
+Something to note, Extended Attributes doesn't mask secret values. We recommend avoiding setting secret values to prevent visibility in the text box and logs. 
+
+<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/extended-attributes.jpg" width="95%" title="Extended Attributes helps users add profiles.yml attributes to dbt Cloud Environment settings using a free form text box." /> <br />
+
+If you're developing in the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) or [orchestrating job runs](/docs/deploy/deployments), Extended Attributes parses through the provided YAML and extracts the `profiles.yml` attributes. For each individual attribute:
+
+- If the attribute exists in another source (such as your project settings), it will replace its value (like environment-level values) in the profile. It also overrides any custom environment variables.
+
+- If the attribute doesn't exist, it will add the attribute or value pair to the profile. 
+
+The following code is an example of the types of attributes you can add in the **Extended Attributes** text box:
+
+```yaml
+snowflake:
+  target: dev
+  outputs:
+    dev:
+      type: snowflake
+      dbname: jaffle_shop      
+      schema: dbt_alice      
+      threads: 4
```

Comment:
That's not a valid example. It should be exactly the following:

```
dbname: jaffle_shop      
schema: dbt_alice      
threads: 4
```

Reply:
fixed that up, thanks @Fleid !